README.md

```diff
@@ -1,2 +1,4 @@
+[![codecov](https://codecov.io/gh/ChauffeurPrive/nestor-api/branch/master/graph/badge.svg)](https://codecov.io/gh/ChauffeurPrive/nestor-api)
+
 # nestor-api
 Nestor API
```

Comment:
I would just keep the title at the top 🙂 
```suggestion
# nestor-api
[![codecov](https://codecov.io/gh/ChauffeurPrive/nestor-api/branch/master/graph/badge.svg)](https://codecov.io/gh/ChauffeurPrive/nestor-api)

Nestor API
```

Reply:
Updated it following React one

Reply:
@benard-g @maximejimenez Finally also added license and workflow badges :)

Reply:
Oh great! I just suggested to add the worflow badge here 😛  https://github.com/ChauffeurPrive/nestor-api/pull/1#issuecomment-644237093

Reply:
Yes and this one use branch query param on the badge so it gets the right status (which is currently not set because we did not build on master)